.github/CONTRIBUTING.md

```diff
@@ -10,6 +10,7 @@ As we must also comply with the upstream Oracle Common Development and Distribut
 ```
 Portions Copyright [2019] Payara Foundation and/or its affiliates
 ```
+You can also follow along with the steps in this video: [https://youtu.be/gkH-wu-bWkE](https://youtu.be/gkH-wu-bWkE)
```

Comment:
Would be much nicer if the link was in-line like so:
`[this video](https://youtu.be/gkH-wu-bWkE).`

Reply:
@Pandrex247 Made requested changes